_layouts/software-details.html

```diff
@@ -388,8 +391,15 @@ <h2>
                 <p>
                   <span class="label">{{ t.software.contract_with }}</span>
                   {% for contractor in page.publiccode.maintenance.contractors %}
-                  <a href="{{ contractor.website }}" class="until">{{ contractor.name }}</a> {{ t.software.until }} {{
-                  contractor.until}}
+                    {% if contractor.website %}
```

Comment:
Personally, I think it might be better to reverse test against emtpy or nil var just in case. Check line 437 for example